src/components/Icon.tsx

```diff
@@ -548,6 +547,30 @@ export const Icon = ({ name, className, size }: IconProps) => {
 					/>
 				</svg>
 			);
+		case "adjust":
```

Comment:
we haven't been updating the icons in storybook. I wonder if it should be in there